lxc/list.go

```diff
@@ -342,14 +343,23 @@ func (c *listCmd) listContainers(d *lxd.Client, cinfos []api.Container, filters
 			}
 			data = append(data, col)
 		}
+		return data
+	}
 
+	switch c.format {
+	case listFormatCSV:
+		w := csv.NewWriter(os.Stdout)
+		w.WriteAll(tableData())
+		if err := w.Error(); err != nil {
+			return err
+		}
+	case listFormatTable:
 		table := tablewriter.NewWriter(os.Stdout)
 		table.SetAutoWrapText(false)
 		table.SetAlignment(tablewriter.ALIGN_LEFT)
 		table.SetRowLine(true)
 		table.SetHeader(headers)
-		sort.Sort(byName(data))
```

Comment:
It was here https://github.com/lxc/lxd/blob/51f6159e9bd2bc98aeb526d595f072d9b2832b03/lxc/list.go#L348